content/index.html.haml

```diff
@@ -98,16 +98,16 @@ homepage: true
 
       - supporters.select{|sponsor| sponsor[:logo]}.each do |sponsor|
         %li
-          %a{:href => sponsor['url'], :target => '_blank', :rel => 'noreferrer noopener'}
+          %a{:href => sponsor['url'], :target => '_blank', :rel => 'no-referrer no opener'}
             %img{:alt => sponsor['name'], :title => sponsor['name'], :src => expand_link("images/sponsors/#{sponsor['logo']}")}/
 
   .supporters
     %p
       %strong
         We thank the following organizations for their support of the Jenkins
-        project through free and/or open source licensing programs.
+        project through free and/or open-source licensing programs.
```

Comment:
```suggestion
        project through free and/or open source licensing programs.
```

Reply:
The Linux Foundation does not hyphenate "open-source".  They list it as "open source".  I don't think we should change to use a hyphenated form in this single location.